src/screens/news-feed-screen/news-feed-screen.spec.js

```diff
@@ -0,0 +1,111 @@
+import {newsFeedScreenDriver} from './news-feed-screen.driver';
+import {Linking} from 'react-native';
+import {UNKNOWN} from '../../strings';
+import dateFormat from 'dateformat';
+import Chance from 'chance';
+
+const chance = Chance();
+
+describe('News Feed Screen', () => {
+  let driver;
+
+  beforeEach(() => {
+    driver = newsFeedScreenDriver();
+  });
+
+  const defaultProps = {
+    fetchCovidNewsAction: () => {},
+    articles: [],
+  };
+
+  it('renders an empty container when articles could not be fetched', () => {
+    driver.setProps(defaultProps);
+
+    expect(driver.container()).toBeDefined();
+    expect(driver.articles().length).toEqual(0);
+  });
+
+  it('renders a single article along with all data associated correctly', () => {
+    const article = buildArticle();
+    const articles = [article];
+    const expectedTitle = article.title;
+    const expectedSourceName = article.source.name;
+    const expectedDate = dateFormat(article.publishedAt);
+    const expectedImageUrl = article.urlToImage ?? undefined;
+    driver.setProps({...defaultProps, articles});
+
+    expect(driver.articles().length).toEqual(1);
+    expect(driver.articleAtIndex(0).title()).toEqual(expectedTitle);
+    expect(driver.articleAtIndex(0).sourceName()).toEqual(expectedSourceName);
+    expect(driver.articleAtIndex(0).timestamp()).toEqual(expectedDate);
+    expect(driver.articleAtIndex(0).imageUri()).toEqual(expectedImageUrl);
+  });
+
+  it('renders multiple different articles retrieved in order', () => {
+    const articles = Array.from(
+      {length: chance.natural({min: 2, max: 10})},
+      () => buildArticle(),
+    );
+    driver.setProps({...defaultProps, articles});
+
+    expect(driver.articles().length).toEqual(articles.length);
+    expect(driver.articleAtIndex(0).title()).toEqual(articles[0].title);
+    expect(driver.articleAtIndex(1).title()).toEqual(articles[1].title);
+    expect(driver.articleAtIndex(0).title()).not.toEqual(
+      driver.articleAtIndex(1).title(),
+    );
+  });
+
+  it('renders an article with fallbacks if properties not present', () => {
```

Comment:
Test is fine, but the naming is not clear, specifically *fallbacks*. The name should be more specific, i.e., saying that ```Should render an article with default (fallback) *values* when properties *are* not present```. Please validate grammar in test descriptions.